docs/markdown/ios/card/card.pins.md

```diff
@@ -1,11 +1,10 @@
 ---
 title: Card.Pins
-description: The Boards preview block utilizes two of the Board and Section components to present a collection of imagery that typically represents Card.Boards.
+description: Card.Pins is a preview block utilizing two of the Pin and Section components to present a collection of imagery.
```

Comment:
The previous copy here was referencing Card.Boards, so seemed to be incorrect. Not sure if this new copy is factually correct